float32 speed
float32 home_offset
---
bool    is_homed
string  error_message
---
float32 elapsed_time
float32 distance_moved